documentation/asciidoc/computers/compute-module/cmio-display.adoc

```diff
@@ -24,15 +24,17 @@ dtoverlay=vc4-kms-dsi-7inch
 ----
 . Reboot your Compute Module with `sudo reboot`. Your device should detect and begin displaying output to your display.
 
-=== Connect a display to DISP0
+=== Connect a display to DISP0/DSI0
 
-To connect a display to DISP0:
+To connect a display to DISP0/DSI0 on CM1, CM3 and CM4 IO boards:
```

Comment:
Do we need a separate set of instructions for CM5 IO board?

Reply:
Hmm, let me ponder my previous change, I wonder if I have got it right.

Reply:
Ah, I could add a CM5 bit in the previous section.